R/Prediction.R

```diff
@@ -195,6 +195,57 @@ makePrediction.CostSensTaskDesc = function(task.desc, row.names, id, truth, pred
 }
 
 #' @export
+makePrediction.ForecastRegrTaskDesc = function(task.desc, row.names, id, truth, predict.type,
+                                               predict.threshold = NULL, y, time, error = NA_character_, dump = NULL) {
+  data = namedList(c("id", "truth", "response", "se"))
+  if (any(class(y) == "matrix"))
```

Comment:
class checks should be done with `inherits(y, "matrix")`  (check this also in other parts of your code)